README.md

```diff
@@ -25,37 +25,57 @@ These directions below assume you are using AWS.  We will include additional clo
 
 ### Collecting Information
 
-Make sure you know what region you're running in (e.g. `us-west-2`).
+The installation script will prompt for several question. It is useful prepare for some of the answers beforehand to expedite the installation process. Here are a few pointers:
 
-### Starting an AWS VM
+* make sure you know what region you're running in (e.g. `us-west-2`)
+* decide whether you want to create an instance for development or production as it might impact the size and therefore the cost of the host virtual machine
+* find out whether your favorite editor is installed on host virtual machine
+* be sure to know how to create a static IP address for your virtual machine (AWS calls this _Elastic IP_)
+* you will be asked to provide an host domain that points to your EC2 instance; at the time installation all you need is a name (i.e., the domain does not need to exist)
```

Comment:
I think I would say "the domain does not have to be configured in route53" or something like that.